### PR TITLE
Add birthday to calendar

### PR DESCRIPTION
**Pull request by Mariam-98**

### CODE REVIEW

**Comment by Mariam-98** [comment]
This PR should be merged after [Fix calendar permission flow #26](https://github.com/SimplyTechnologies/android-internship/pull/26)

File: app/src/main/java/com/simply/birthdayapp/presentation/ui/screens/main/home/birthday/BithdayScreen.kt
Comment (Hayk985) [inline app/src/main/java/com/simply/birthdayapp/presentation/ui/screens/main/home/birthday/BithdayScreen.kt:1]: Move extension functions out of the composable file to another file.

File: app/src/main/java/com/simply/birthdayapp/presentation/ui/screens/main/home/birthday/BirthdayViewModel.kt [hunk 1]
```diff
@@ -170,7 +214,10 @@ class BirthdayViewModel(
         viewModelScope.launch(Dispatchers.IO) {
             birthdayRepository.deleteBirthday(id)
                 .onEach {
-                    it.onSuccess { navigateToHomeScreen() }
+                    it.onSuccess {
+                        navigateToHomeScreen()
```
Comment (Hayk985) [inline app/src/main/java/com/simply/birthdayapp/presentation/ui/screens/main/home/birthday/BirthdayViewModel.kt]: This maybe already fixed in other PR, but no need to pass this to `ViewModel`. Use flow instead.

File: app/src/main/java/com/simply/birthdayapp/presentation/ui/screens/main/home/birthday/BithdayScreen.kt [hunk 2]
```diff
@@ -270,9 +314,40 @@ fun BirthdayScreen(
...
+                        birthdayViewModel.setAddToCalendarCheck(it)
+                    },
+                    colors = CheckboxDefaults.colors(
+                        uncheckedColor = AppTheme.colors.lightPink,
+                        checkedColor = AppTheme.colors.lightPink,
+                    ),
+                )
+                Text(
+                    modifier = Modifier
+                        .fillMaxHeight(),
```
Comment (Hayk985) [inline app/src/main/java/com/simply/birthdayapp/presentation/ui/screens/main/home/birthday/BithdayScreen.kt]: Write in a single line

**Review by Hayk985 — APPROVED** [review]

**Pull request merged**